upgrade/capsule.py

```diff
@@ -88,15 +145,17 @@ def satellite6_capsule_setup(sat_host, os_version, upgradable_capsule=True):
     copy_ssh_key(sat_host, cap_hosts)
     # Dont run capsule upgrade requirements for n-1 capsule
     if upgradable_capsule:
+        save_server_config(sat_host)
```

Comment:
One time initialization as suggested above.

Reply:
no need to put it as global, it dumps an error message, because the satellite host is not static.